src/mgmt/srv_target.c

```diff
@@ -1180,7 +1180,7 @@ ds_mgmt_hdlr_tgt_create(crt_rpc_t *tc_req)
 		/* A zero size accommodates the existing file */
 		vpa.vpa_scm_size = 0;
 		vpa.vpa_nvme_size = tc_in->tc_nvme_size / dss_tgt_nr;
-		rc = dss_thread_collective(tgt_vos_create_one, &vpa, 0);
+		rc = dss_thread_collective(tgt_vos_create_one, &vpa, DSS_ULT_DEEP_STACK);
```

Comment:
This change should probably be reviewed by someone familiar with the repercussions on the engine side. Maybe @NiuYawei ?

Reply:
this looks fine.  It just uses deeper stack by default for creating the vos target.

Reply:
Yes, we've already used deep stack size for regular pool open/create ULTs in other places, this is a missed case.